### PR TITLE
bugfix: `@Multiple` can contain scalar types

### PR DESCRIPTION
**Pull request by t1**
I stumbled over this issue when I tried to request multiple mutations in one

### CODE REVIEW

File: client/api/src/main/java/io/smallrye/graphql/client/typesafe/api/Multiple.java [hunk 1]
```diff
@@ -38,6 +38,8 @@
  *   }
  * }
  * </pre>
+ * <p>
```
Comment (phillip-kruger) [inline client/api/src/main/java/io/smallrye/graphql/client/typesafe/api/Multiple.java:41]: Should this not be </p> ?

File: client/api/src/main/java/io/smallrye/graphql/client/typesafe/api/Multiple.java [hunk 2]
```diff
@@ -25,9 +25,9 @@
  *     Bar bar;
  * }
  * </pre>
- *
+ * <p>
```
Comment (phillip-kruger) [inline client/api/src/main/java/io/smallrye/graphql/client/typesafe/api/Multiple.java:28]: Should this not be </p> ?

Comment (t1) [reply client/api/src/main/java/io/smallrye/graphql/client/typesafe/api/Multiple.java]: Maybe your formatting got mixed up, so I'm not sure what you meant. Maybe this is better?

**Review by phillip-kruger — APPROVED** [review]
@jmartisk - Please have a look and merge if you are happy

**Review by jmartisk — APPROVED** [review]

**Pull request merged**